<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startup</class>
 <widget class="QWidget" name="startup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AirClip</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>500</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <widget class="QGroupBox" name="SignUpPage_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="title_main">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>311</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>72</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>AirClip</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="MainLoginButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>300</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="MainSignupButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>300</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
      <widget class="QLabel" name="WelcomeText">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>511</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Welcome to AirClip! If you are a new user you can create an account by clicking on Sign Up. If you already have an account then proceed to Login.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <widget class="QGroupBox" name="LoginPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="title_login">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>311</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>72</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>AirClip</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="UserNLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>180</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="PassLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>235</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LoginPageUsername">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>180</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50);
selection-background-color: rgb(50, 50, 50);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="LoginPagePassword">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>230</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="LoginPageLoginButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>310</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LoginPageBackButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>310</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <widget class="QGroupBox" name="SignUpPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="title_signup">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>311</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>72</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>AirClip</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="UserNLabel_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>180</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="PassLabel_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>235</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="SignUpUsername">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>180</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50);
selection-background-color: rgb(50, 50, 50);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="SignUpPassword">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>230</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="SignUpPageSignUpButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>380</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SignUpPageBackButton">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>380</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="SignUpPassword_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>280</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="PassLabel_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>270</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Confirm password</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="SignUpErrorLabel">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>340</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <widget class="QGroupBox" name="SignUpPage_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="title_scanner">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>311</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>72</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>AirClip</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="scannerReload">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style:solid;
background-color: rgb(216, 216, 216);
border-width:1px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Refresh List</string>
       </property>
      </widget>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>421</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);
border-style:solid;
border-width:2px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="scannerConnect">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>430</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style:solid;
background-color: rgb(216, 216, 216);
border-width:1px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="selectedLabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>380</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Selected:</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
